governance/default.go

```diff
@@ -401,6 +402,23 @@ func (gs *GovernanceSet) Merge(change map[string]interface{}) {
 }
 
 func NewGovernance(chainConfig *params.ChainConfig, dbm database.DBManager) *Governance {
+	return &Governance{
+		ChainConfig:              chainConfig,
+		voteMap:                  NewVoteMap(),
+		db:                       dbm,
+		itemCache:                newGovernanceCache(),
+		currentSet:               NewGovernanceSet(),
+		changeSet:                NewGovernanceSet(),
+		lastGovernanceStateBlock: 0,
+		GovernanceTallies:        NewGovernanceTallies(),
+		GovernanceVotes:          NewGovernanceVotes(),
+		idxCacheLock:             new(sync.RWMutex),
+	}
+}
+
+// NewGovernanceInitialize creates Governance with the given configuration and read governance state from DB.
+// It also write governance items of the genesis block to DB if any items are stored in DB.
+func NewGovernanceInitialize(chainConfig *params.ChainConfig, dbm database.DBManager) *Governance {
 	ret := Governance{
```

Comment:
```suggestion
	governance := *NewGovernance(chainConfig, dbm)
```
How about calling the function inside?